mongodb/map.jinja

```diff
@@ -1,14 +1,19 @@
 {% set mongodb = salt['grains.filter_by']({
     'default': {
-        'pkg': 'mongodb',
-        'service': 'mongodb',
-        'conf_file': '/etc/mongodb/mongodb.conf',
-        'config': {
-            'key': 'value',
-        },
+      'key': '7F0CEB10',
+      'keyserver': 'keyserver.ubuntu.com',
+      'pkgs': [
+        {'mongodb-10gen': '*'},
```

Comment:
I've fixed the representation, but the package is `mongodb-10gen` for both RedHat and Debian, so I've left it in `default`.